docs/_snippets/storybook-manager-sidebar-hide-on-landing.md

```diff
@@ -0,0 +1,16 @@
+```js filename="./storybook/manager.js" renderer="common" language="js"
+import { addons, type State } from '@storybook/manager-api';
+
+addons.setConfig({
+  layoutCustomisations: {
+    // Hide the sidebar on the landing page, which has its own nav links to other pages.
+    showSidebar(state: State, defaultValue: boolean) {
+      if (state.storyId === 'landing' && state.viewMode === 'docs') {
+        return false;
+      }
+
+      return defaultValue;
+    },
+  },
+});
+```
```

Comment:
@Sidnioulz would you be ok with adjusting this and the other example to be "pure" JavaScript and include a TypeScript variant (i.e., remove the `Type` definition) to avoid confusion

Reply:
Sure! Should I put two code blocks in the same MD file? Is there a specific technique to apply for the docs UI to link two code snippets with different languages?

Reply:
This should now be addressed!